qubesmanager/qube_manager.py

```diff
@@ -852,6 +863,19 @@ def action_clonevm_triggered(self):
         if not ok or clone_name == "":
             return
 
+        name_in_use = False
+        # check if VM name is already in use:
+        for domain in self.qubes_app.domains:
+            if domain.name == clone_name:
+                name_in_use = True
```

Comment:
Better check for existence in collection, not iterate over all domains: `clone_name in self.qubes_app.domains`